packages/slice-machine/lib/mock/Slice.ts

```diff
@@ -0,0 +1,58 @@
+import * as Widgets from "./misc/widgets";
```

Comment:
Like that

Reply:
I didn't fix that one, I fixed the other one first.